tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -1512,26 +1455,35 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 		pvcName := "cns-pvc-" + curtimeinstring
 		framework.Logf("pvc name :%s", pvcName)
 
-		restConfig, storageclass, profileID := staticProvisioningPreSetUpUtil(ctx)
+		k8senv := GetAndExpectStringEnvVar("KUBECONFIG")
+		restConfig, err := clientcmd.BuildConfigFromFlags("", k8senv)
```

Comment:
consider using `framework.ClientConfig()`

Reply:
Tried to use framework.ClientConfig() , this is causing multiple errors, 
Instead replaced the code with getRestConfigClient() method